client/pages/index.js

```diff
@@ -31,7 +31,7 @@ function App({ users }) {
 
 App.getInitialProps = async () => {
   try {
-    const { data } = await axios.get('http://localhost:8080/api/user')
+    const { data } = await axios.get('https://api-dot-resume-engine.appspot.com/api/user')
```

Comment:
Why the second app engine application (the url)? 🤔

Reply:
I wasn't able to figure out how to connect React to the Golang server it was hosted in. I wasn't even able to build React for some reason...